docs/src/main/tut/auth.md

```diff
@@ -101,15 +102,17 @@ val logIn: Service[Request, Response] = Kleisli({ request =>
 Now that the cookie is set, we can retrieve it again in the `authUser`.
 
 ```tut:book
+import fs2.interop.cats._ 
+
 def retrieveUser: Service[Long, User] = Kleisli(id => Task.delay(???))
-val authUser: Service[Request, String \/ User] = Kleisli({ request =>
+val authUser: Service[Request, String Either User] = Kleisli({ request =>
   val message = for {
-    header <- headers.Cookie.from(request.headers).toRightDisjunction("Cookie parsing error")
-    cookie <- header.values.list.find(_.name == "authcookie").toRightDisjunction("Couldn't find the authcookie")
-    token <- crypto.validateSignedToken(cookie.content).toRightDisjunction("Cookie invalid")
-    message <- \/.fromTryCatchNonFatal(token.toLong).leftMap(_.toString)
+    header <- headers.Cookie.from(request.headers).toRight("Cookie parsing error")
+    cookie <- header.values.toList.find(_.name == "authcookie").toRight("Couldn't find the authcookie")
+    token <- crypto.validateSignedToken(cookie.content).toRight("Cookie invalid")
+    message <- Either.catchOnly[NumberFormatException](token.toLong).leftMap(_.toString)
   } yield message
-  message.traverse(retrieveUser)
+  message.traverse(retrieveUser.run)
```

Comment:
We're going to go with this for the time being until we discover a better option